vroom/neovim_mod.py

```diff
@@ -1,13 +1,16 @@
 from vroom.vim import CONFIGFILE
 from vroom.vim import Communicator as VimCommunicator
 import subprocess
+import tempfile
 import time
 import neovim
 import os
 
 class Communicator(VimCommunicator):
   """Object to communicate with a Neovim server."""
 
+  _listen_addr: 'Optional[str]' = None
```

Comment:
I don't think we have typing checks enabled, so until then I'd prefer no type annotations.

Reply:
No problem, updated.